CODE_OF_CONDUCT.md

```diff
@@ -63,6 +63,15 @@ Project maintainers who do not follow or enforce the Code of Conduct in good
 faith may face temporary or permanent repercussions as determined by other
 members of the project's maintenance team.
 
+### Code of Conduct Committee
+
+The current Code of Conduct Committee consists of:
+
+- Neil Chue Hong ([N.ChueHong@epcc.ed.ac.uk](mailto:N.ChueHong@epcc.ed.ac.uk))
+- Robert Haines ([robert.haines@manchester.ac.uk](mailto:robert.haines@manchester.ac.uk))
+
+You may contact members individually or collectively. 
```

Comment:
Do we need to add a clarification here that even if members are contacted individually, then others will be involved in resolving the issue (unless the complaint is raised against a member of the CoC, of course)?

Reply:
Thanks @hainesr, good point.
Can you check if https://github.com/citation-file-format/citation-file-format/pull/494/commits/c492b16934fc9b25b9ff1426909df9cc5a635d0a resolves this for you?

Reply:
Looks good to me, thanks!